packages/backend/codegen.yml

```diff
@@ -8,7 +8,7 @@ generates:
     config:
       defaultMapper: Partial<{T}>
       # By default it is T | null. But the service code uses typescript's optional types (`foo?: T`) which are `T | undefined`. This saves the trouble of converting them explicitly.
-      maybeValue: T | undefined
+      inputMaybeValue: T | undefined
```

Comment:
If we were to just remove this altogether would `inputMaybeValue` default to `T | undefined` already?

Reply:
Previously, we had
https://github.com/interledger/rafiki/blob/c10b56e8bbf869f3c9e08df84946b20a6ae45e70/packages/backend/src/graphql/generated/graphql.ts#L2-L3
This pr changed it to
```
export type Maybe<T> = T | null;
export type InputMaybe<T> = T | undefined;
```
Removing `inputMaybeValue` makes it
```
export type Maybe<T> = T | null;
export type InputMaybe<T> = Maybe<T>;
```
but this seems to work without build/test issues even though we've got inputs throughout resolver tests with undefined fields :confused: 
(I haven't tried removing `inputMaybeValue` with strict mode enabled)

I'm ok just removing `inputMaybeValue`